packages/webdriverio/src/scripts/resq.js

```diff
@@ -0,0 +1,38 @@
+export const waitToLoadReact = function waitToLoadReact () {
+    window.resq.waitToLoadReact()
+}
+
+export const react$ = function react$ (selector, props) {
+    const elems = window.resq.resq$$(selector)
+
+    if (Object.keys(props).length === 0) {
+        return elems[0]
+    }
+
+    let elem = [...elems].find((elem) => {
+        for (const [name, value] of Object.entries(props)) {
+            // logs.push(`check ${name} and ${value} for ${elem[name]}`)
+            if (elem.props[name] === value) {
+                return true
+            }
+        }
+
+        return false
+    })
+
+    if (!elem || (!elem.node && elem.children.length === 0)) {
+        return
+    }
+
+    if (elem.node) {
+        return elem.node
+    }
+
+    elem = elem.node || elem.children[0].node
```

Comment:
As I mentioned to you on gitter. This will not quite work as expected if the JSX is wrapped in a Fragment.

With this implementation, this is how it'll work:

```javascript
import React from 'react';

const MyCmp = () => (
  <React.Fragment>
    <div>something</div>
    <div> something else </div>
  </React.Fragment>
)


//

react$('MyCmp') // <div>something</div>
```

Reply:
I'm working on a feature to fix this, it's still raw and untested, but here's the gist of it https://github.com/baruchvlz/resq/commit/7b9ffd1272da79ec4ace45768ff0282e93fcad5f